leshan-lwm2m-core/src/main/java/org/eclipse/leshan/senml/SenMLPack.java

```diff
@@ -47,7 +47,7 @@ public void addRecords(List<SenMLRecord> records) {
     public List<SenMLRecord> getRecords() {
         if (records == null)
             return Collections.emptyList();
-        return records;
```

Comment:
If record in modifiable why remake it unmodifiable on getter ? :thinking:

Reply:
OK - corrected